Controller/Controller.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\PageBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller as BaseController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Finder\Finder;
+
+use Integrated\Bundle\ContentBundle\Document\Channel\Channel;
+
+/**
+ * @author Ger Jan van den Bosch <gerjan@e-active.nl>
+ */
+class Controller extends BaseController
```

Comment:
Instead of extending i'd rather see this split into services and classes (controllers as service).